cmake/cmake_options.yml

```diff
@@ -46,7 +46,8 @@ USE_SIGNAL_HANDLER: "ON" # Print stack traces on segfaults.
 USE_TENSORRT: "OFF" # Enable infeference optimization with TensorRT.
 USE_ASAN: "OFF" # Enable Clang/GCC ASAN sanitizers.
 ENABLE_TESTCOVERAGE: "OFF" # Enable compilation with test coverage metric output
-CMAKE_BUILD_TYPE: "Debug"
+USE_INT64_TENSOR_SIZE: "OFF" # Use int64_t to represent the total number of elements in a tensor
+CMAKE_BUILD_TYPE: "Debug" # Debug | Release | RelWithDebInfo | MinSizeRel
```

Comment:
The danger of this default option is that users are not aware they are building a Debug instead of Release.

Reply:
This file is only used for the dev_menu.py script build. If you call cmake explicitly it doesn't affect the cmake command.

Reply:
I don't care about Debug or Release, I assume that if someone is building they want to develop or diagnose a problem. For developers I think most of the time Debug is the build you want.